config.example/group_vars/slurm-cluster.yml

```diff
@@ -91,6 +91,9 @@ slurm_enable_nfs_client_nodes: true
 nfs_server_group: "slurm-master[0]"
 nfs_client_group: "slurm-master[1:],slurm-node"
 
+# Flags for sharing slurm.conf among compute and control nodes
+slurm_conf_symlink: false
```

Comment:
Can you duplicate this line in `roles/slurm/defaults/main.yml` as well as well?

This will make sure the role itself is functional, even if this line isn't present in a cluster config. This will impact upgrades of older clusters and some of our CI workflows.

Reply:
Flag is now placed in that file for the same.